.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: freertos-pkcs11-psa CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    name: Build cypress/CY8CKIT_064S0S2_4343W
+    runs-on: ubuntu-latest
+    steps:
+    - name: Clone
+      uses: actions/checkout@v1
+    - name: Install gnuarmemb
+      run: |
+        mkdir -p ~/bin
+        mkdir -p ~/opt
+        cd ..
+        wget -q https://armkeil.blob.core.windows.net/developer/Files/downloads/gnu-rm/9-2019q4/RC2.1/gcc-arm-none-eabi-9-2019-q4-major-x86_64-linux.tar.bz2
```

Comment:
At some point we should look at using a docker container, but this seems fine for now.